### PR TITLE
SFR-495 Titles need to follow UX documentation

### PR DESCRIPTION
**Pull request by nachocs**
Also solved bug when going back to home that didn't remove results.
And improvement on isEmpty util that wasn't working 100%

### CODE REVIEW

File: src/app/components/SearchContainer/SearchContainer.jsx [hunk 1]
```diff
@@ -29,6 +29,7 @@ class SearchContainer extends React.Component {
 
   componentDidMount() {
     this.loadSearch();
+    global.document.title = 'Search Results | ResearchNow | NYPL';
```
Comment (mwbenowitz) [inline src/app/components/SearchContainer/SearchContainer.jsx]: This is getting set when I first load the site. Is there any way that the `SearchContainer` component is mounting before the page loads and then is not rendered?

Comment (nachocs) [reply src/app/components/SearchContainer/SearchContainer.jsx]: No, but I noticed there is a mistake going back to home. Fixing it now.

File: src/app/components/SearchContainer/SearchContainer.jsx [hunk 2]
```diff
@@ -37,6 +38,18 @@ class SearchContainer extends React.Component {
...
     }
+    this.setTitle();
+  }
+
+  setTitle() {
+    const {
+      location: { query },
+    } = this.props;
+    if (!query || isEmpty(query)) {
+      global.document.title = 'ResearchNow';
```
Comment (mwbenowitz) [inline src/app/components/SearchContainer/SearchContainer.jsx]: I think that this should be "ResearchNow | NYPL"?

**Comment by nachocs** [comment]
I refactored all this, using a common const and applying the title in the Application instead of components, please review again

File: src/app/components/SearchContainer/SearchContainer.jsx (hunk 2 above)
Comment (nachocs) [reply src/app/components/SearchContainer/SearchContainer.jsx]: That's not approved yet, I think

**Pull request merged**